_data/glossary.yaml

```diff
@@ -170,6 +175,12 @@ Machine: |
   installs Docker on them, then configures the Docker client to talk to them.
 
   *Also known as : docker-machine*
+namespace: |
```

Comment:
Actually Docker namespaces are not quite the same as Linux kernel namespaces. I'll help out with this definition.